Exercises-1-Basic/src/exercises/Number6Shape2.java

```diff
@@ -0,0 +1,25 @@
+package exercises;
+
+/*
+   ii)
+
+*
+**
+***
+****
+*****
+ */
+public class Number6Shape2 {
```

Comment:
You should rename some class names.
This class is called `Number6Shape2`, but it is actually `Number5Shape2`, right?
It is still part of Number5 exercise, just a different shape.

When you rename the class, make sure to use Eclipse's refactoring.

Reply:
Not done :-(

Reply:
Done by refactored all shape files(number5 shape1,2,3,4,5,6)